analyzers/tests/SonarAnalyzer.UnitTest/Rules/CollectionEmptinessCheckingTest.cs

```diff
@@ -31,6 +31,17 @@ public class CollectionEmptinessCheckingTest
         [TestMethod]
         [TestCategory("Rule")]
         public void CollectionEmptinessChecking() =>
-            Verifier.VerifyAnalyzer(@"TestCases\CollectionEmptinessChecking.cs", new CollectionEmptinessChecking());
+            Verifier.VerifyAnalyzer(
+                @"TestCases\CollectionEmptinessChecking.cs",
+                new CollectionEmptinessChecking());
```

Comment:
This one could remain as single liner

Reply:
Sure. No guidelines of making lines too long?

Reply:
S103 will tell you :) It's configured for 200 characters.